mdstat.go

```diff
@@ -165,6 +175,10 @@ func parseMDStat(mdStatData []byte) ([]MDStat, error) {
 	return mdStats, nil
 }
 
+func isRaidType(mdType string) bool {
+	return strings.HasPrefix(mdType, "raid") || mdType == "linear"
```

Comment:
No, I don't think this is going to work. Like I said, there are a whole bunch of other types other than `raid...`. Since this can change depending on the kernel/mdadm version. We can't support an exhaustive list here.